test/unit/theory/theory_engine_white.h

```diff
@@ -91,118 +92,140 @@ struct RewriteItem {
   bool d_topLevel;
 };/* struct RewriteItem */
 
+class FakeTheoryRewriter : public TheoryRewriter
+{
+ public:
+  /**
+   * Overrides TheoryRewriter::preRewrite().  This "fake theory" version
+   * ensures that this actual, observed pre-rewrite call matches the next
+   * "expected" call set up by the test.
+   */
+  RewriteResponse preRewrite(TNode n)
+  {
+    //    if(false) { //s_expected.empty()) {
```

Comment:
Do we need this? Perhaps delete this and the other comment blocks.  I assume this is broken code?